tests/kafkatest/services/verifiable_producer.py

```diff
@@ -68,9 +98,30 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+                        # Log information if there is a large gap between successively acknowledged messages
+                        t = time.time()
+                        time_delta_sec = t - last_produced_time
+                        if time_delta_sec > 2 and prev_msg is not None:
```

Comment:
I'm not sure if it does or not. Perhaps this can be made customizable or removed if the need arises
We may even want to remove it once we sort out the bug causing these large deltas